app-engine/src/main/java/be/cytomine/appengine/services/TaskService.java

```diff
@@ -199,7 +201,7 @@ public Optional<TaskDescription> uploadTask(InputStream inputStream)
             log.error("UploadTask: task already exists");
             throw e;
         } catch (Exception e) {
-            log.error("UploadTask: Unknown bundle archive format");
+            log.error(format("UploadTask: Unknown bundle archive format %s", e));
```

Comment:
```suggestion
            log.error("UploadTask: Unknown bundle archive format {}", e);
```

And the `import static java.lang.String.format;` can also be removed